application/src/sass/components/_table.scss

```diff
@@ -93,11 +93,15 @@
 
 /* Hide the main table header content (in case it peeks through
    when bouncing back from momentum scroll) */
-.table-container thead th, .table-container thead td {
+.eff-table__head--transparent, .eff-table__caption--transparent {
   opacity: 0;
   border-color: transparent;
 }
 
+.eff-table__caption {
+  display: table-caption;
```

Comment:
Not sure we need this? Should be the default...?

Reply:
It gets overriden by `govuk-heading-s`; it seemed like we needed it.